charts/tidb-operator/templates/advanced-statefulset-deployment.yaml

```diff
@@ -0,0 +1,55 @@
+{{- if .Values.advancedStatefulset.create }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: advanced-statefulset-controller
+  labels:
+    app.kubernetes.io/name: {{ template "chart.name" . }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/component: advanced-statefulset-controller
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+"  "_" }}
+spec:
+  replicas: {{ .Values.advancedStatefulset.replicas }}
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ template "chart.name" . }}
+      app.kubernetes.io/instance: {{ .Release.Name }}
+      app.kubernetes.io/component: advanced-statefulset-controller
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: {{ template "chart.name" . }}
+        app.kubernetes.io/instance: {{ .Release.Name }}
+        app.kubernetes.io/component: advanced-statefulset-controller
+    spec:
+      serviceAccountName: advanced-statefulset-controller
+      containers:
+      - name: advanced-statefulset-controller
+        image: {{ .Values.advancedStatefulset.image }}
+        imagePullPolicy: {{ .Values.advancedStatefulset.imagePullPolicy | default "IfNotPresent" }}
+        args:
+        - --v=4
```

Comment:
The log level of all components we deployed is set to 2 by default. Is it set to 2 by default?

Reply:
fixed, add a logLevel field to specify log level (defaults to 2) same as tidb-controller-manager